Makefile

```diff
@@ -269,4 +269,9 @@ check-links:
 	docker build -t check-links scripts/docs/check-links
 	docker run --rm -v $(CURDIR):/usr/src/signalfx-agent:ro check-links
 
+.PHONY: dependency-check
+dependency-check: BUNDLE_PATH ?= signalfx-agent-$(shell ./scripts/current-version).tar.gz
+dependency-check:
```

Comment:
I wanted `BUNDLE_PATH` to be a target-specific variable, otherwise we'll get this error:
```
Makefile:274: *** commands commence before first target.  Stop.
```